tin/templates/assignments/quiz.html

```diff
@@ -22,7 +26,12 @@
 
           var editor = ace.edit("editor");
           editor.setTheme("ace/theme/tomorrow");
-          editor.session.setMode("ace/mode/python");
+
+          var modelist = ace.require("ace/ext/modelist");
+          var filename = "{{ assignment.filename }}";
+          var mode = modelist.getModeForPath(filename).mode;
+          console.log(mode);
```

Comment:
Maybe make this more descriptive? Something like `"Using mode "+mode+" for LSP"`?